src/components/DatePicker/DatePickInput/DatePickInput.tsx

```diff
@@ -13,12 +16,16 @@ const ON_CHANGE_MOCK = () => {};
 
 type Props = {
   handleFocus: () => void;
-  handleClear: (event: React.KeyboardEvent) => void;
+  handleClear: (event?: React.KeyboardEvent) => void;
+  filterType?: FilterType;
   isRangePicker: boolean;
   selectedDay: Range;
   /** Props for styling the input */
   inputProps?: TextFieldProps;
   dateFormatOverride?: DateFormatType;
+  open: boolean;
+  buttonType?: 'primary' | 'secondary';
```

Comment:
i am very skeptical regarding these props `buttonType, styleType, filterType` as these are referring only to filter. I would suggest having them as one object that have them all. 

Any ideas here @elektracodes @Christos-Zacharopoulos ?

Reply:
another approach could be to have a context on the `FilterBase` to pass these values and then `Filter`, `Datepicker` and `DatePickInput` will read these values if under that context from provider. Let me know your thoughts @Orfium/front-end

Reply:
I feel like adding a context would be too much to accomplish what we want. The object idea sounds good to me!

Reply:
Sorry for being late. I was on PTO.

I prefer the object as prop. If we go to a context solution I would prefer to consider it after global states are implemented, as ther context described above is a kind of global behavior thingy thing.

Something like `filterBaseProps` like material does with it's components.